cli/src/ios/build.ts

```diff
@@ -45,12 +45,23 @@ export async function buildiOS(config: Config, buildOptions: BuildCommandOptions
     ),
   );
 
+  const manualSigningContents = `<key>provisioningProfiles</key>
+<dict>
+<key>${config.app.appId}</key>
+<string>${buildOptions.xcodeProvisioningProfile ?? ''}</string>
+</dict>
+<key>signingCertificate</key>
+<string>${buildOptions.xcodeSigningCertificate ?? ''}</string>`;
```

Comment:
Will these be OK as empty strings, or would it fail the build?

Reply:
Yeah, that block is only added if signing style is manual, and if that's chosen, presumably, the developer would be supplying a provisioning profile and/or signing certificate.  If what they supply is wrong (or if it's empty), then it won't work.

I believe the signing certificate can be optional, so I may push up a change to remove that key and string if the value is empty.